ASIM/dev/ASimTester/ASimTester.csv

```diff
@@ -667,7 +667,7 @@ EventStartTime,datetime,Mandatory,NetworkSession,,,
 EventSubType,string,Optional,NetworkSession,Enumerated,Start|End|,
 EventType,string,Mandatory,NetworkSession,Enumerated,NetworkSession|L2NetworkSession|EndpointNetworkSession|Flow,
 EventUid,string,Recommended,NetworkSession,,,
-EventVendor,string,Mandatory,NetworkSession,Enumerated,Fortinet|AppGate|Palo Alto|Microsoft|Zscaler|AWS|Vectra AI|WatchGuard|Cisco|Corelight|Check Point,
+EventVendor,string,Mandatory,NetworkSession,Enumerated,Fortinet|AppGate|Palo Alto|Microsoft|Zscaler|AWS|Vectra AI|WatchGuard|Cisco|Corelight|Check Point|FORCEPOINT,
```

Comment:
Why all capital?

Reply:
The source is sending it in this format.

Reply:
Sure. This is why I asked you to use a static value in the parser as well... we are here to fix source mischievous behavior :-)

Reply:
Understood :-)
I have updated in the Parser as well.

Reply:
Did you push for this PR? I don't see the change here yet.

Reply:
I have pushed the changes just now. Also, added ForcepointId as well in DvcIdType as it part of the similar effort.